t3/simulate/rmg_simulator.py

```diff
@@ -0,0 +1,273 @@
+                elif isinstance(reaction_system, LiquidReactor):
+                    reaction_system.sensitive_species = self.observable_species
+                    reaction_system.sensitivity_threshold = self.t3['sensitivity']['SA_threshold']
+                    if hasattr(reaction_system, 'Trange') and reaction_system.Trange is not None:
+                        temperature = sum([t.value_si for t in reaction_system.Trange]) / len(reaction_system.Trange)
+                    else:
+                        temperature = reaction_system.T.value_si
+                    reaction_system.sens_conditions['T'] = temperature
+                    if hasattr(reaction_system, 'Vrange') and reaction_system.Vrange is not None:
+                        volume = sum([v for v in reaction_system.Vrange]) / len(reaction_system.Vrange)
+                    else:
+                        volume = reaction_system.V
+                    reaction_system.sens_conditions['V'] = volume
+                else:
+                    raise NotImplementedError(f'RMG SA not implemented for Reactor type {type(reaction_system)}.')
+
+        try:
+            self.logger.info('Simulating a job using RMG...')
+            simulate(self.rmg_model)
+        except FileNotFoundError:
+            self.logger.error('The RMG adapter did not properly simulate the job.')
+
+
+    def get_sa_coefficients(self):
+        """
+        Obtain the SA coefficients.
+
+        Returns:
+             A SA dictionary, whose structure is given below.
+             sa_dict = { 'thermo' :
+                                    { observable_1 :
+                                                    { species_1 : 1D array with one entry per time point. Each entry
+                                                                  is dLn(observable_1) / dG_species_1 in mol / kcal
+                                                                  at the respective time.
+
+                                                     continues for all other species in the model i.e species_2, etc.
+
+                                                    }
+                                      observable_2 :  etc...
+                                    }
+
+                         'kinetcs' :
+                                    { observable_1 :
+                                                    { 1     :   1D array with one entry per time point. Each entry
+                                                                is dLn(observable_1) / dLn(k_1) at the respective time.
+
+                                                     continues for all other reactions in the model i.e. 2, 3, etc.
+
+                                                    }
+                                      observable_2 :  etc...
+                                    }
+
+                         'time' :  1D array of time points in seconds.
+                        }
+        """
+
+        def get_species_by_label(label: str,
```

Comment:
How is this different than t3.main.get_species_by_label()? Maybe move the original one to common and use it here as well?